src/core/urdb.jl

```diff
@@ -229,7 +229,7 @@ function parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM
         energy_tier_max = get(energy_tier, "max", bigM)
 
         if "rate" in keys(energy_tier) || "adj" in keys(energy_tier)  || "sell" in keys(energy_tier)
-            append!(energy_tier_limits_kwh, energy_tier_max)
+            append!(energy_tier_limits_kwh, min(energy_tier_max, bigM))
```

Comment:
Do you think we should consider increasing the bigM default in `parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM = 1.0e8)`? Something like a big steel plant might use more than 1.0e8 kWh per year

Reply:
Good call!  Noting here that after a quick sidebar with you I've increased it to 1e10 for now.  Longer term, we should try to get the loads as input so that we can get more accurate big-M's (we can use the max monthly energy demand for example) - I'll add an issue to that effect.

Reply:
> Do you think we should consider increasing the bigM default in `parse_urdb_energy_costs(d::Dict, year::Int; time_steps_per_hour=1, bigM = 1.0e8)` and `parse_urdb_demand_tiers`? Something like a big steel plant might use more than 1.0e8 kWh per year

addressed here: https://github.com/NREL/REopt.jl/commit/0c5207fe1c65d671056078bd6e2c8ce890df41c3

Reply:
Looks good! Just documenting that I should have said "per month" and that this change only applies to the energy tiers bigM, not the demand tiers.